Resources/Prototypes/Traits/disabilities.yml

```
@@ -30,7 +30,7 @@
   id: Colorblindness
   name: Colorblindness
   description: You are only able to see certain colors, similarly to a dog's vision.
-  cost: 3
+  cost: 2
```

```suggestion
  cost: 3
```
This is legitimately worse than monochromacy :P
It's not like, slightly more colour equal slightly more better, this one skews your whole vision whereas monochromacy is still representative of the world.